lib/elasticity/search.rb

```diff
@@ -243,6 +217,26 @@ def pretty_print(pp)
           end
         end
 
+        def total
+          response["hits"]["total"]
+        end
+
+        # for pagination
+        def per_page
+          @search_definition.body[:size] || DEFAULT_SIZE
+        end
+
+        # for pagination
+        def total_pages
+          (total.to_f / per_page.to_f).ceil
+        end
+
+        # for pagination
+        def current_page
+          return 1 if @search_definition.body[:from].nil?
+          @search_definition.body[:from] / per_page + 1
+        end
+
```

Comment:
Any reason why internally you can't re-use the Results objects in order to delegate these methods to?